next/src/components/_global/tableOfContent/tableOfContent.module.scss

```diff
@@ -3,7 +3,7 @@
   position: sticky;
   top: 75px;
   overflow-y: auto;
-  max-height: calc(100dvh - 75px);
+  max-height: calc(80dvh - 75px);
```

Comment:
Specify fallback for the `dvh`:
```
max-height: calc(80vh - 75px);
max-height: calc(80dvh - 75px);
```
Remember about that, and change that globally if it exist somewhere else.
Can I Use: https://caniuse.com/?search=dvh